python/ray/projects/projects.py

```diff
@@ -0,0 +1,103 @@
+import json
+import jsonschema
+import os
+import yaml
+
+
+def find_root(dir):
```

Comment:
Rename `dir -> directory` (`dir` is a builtin method in python).

Reply:
fixed